starboard/file.h

```diff
@@ -129,38 +129,7 @@ static inline bool SbFileIsValid(SbFile file) {
   return file != kSbFileInvalid;
 }
 
-// DEPRECATED with SB_API_VERSION 16
-//
-// Opens the file at |path|, which must be absolute, creating it if specified by
-// |flags|. The read/write position is at the beginning of the file.
-//
-// Note that this function only guarantees the correct behavior when |path|
-// points to a file. The behavior is undefined if |path| points to a directory.
-//
-// |path|: The absolute path of the file to be opened.
-// |flags|: |SbFileFlags| that determine how the file is used in the
-//   application. Among other things, |flags| can indicate whether the
-//   application should create |path| if |path| does not already exist.
-// |out_created|: Starboard sets this value to |true| if a new file is created
-//   or if an old file is truncated to zero length to simulate a new file,
-//   which can happen if the |kSbFileCreateAlways| flag is set. Otherwise,
-//   Starboard sets this value to |false|.
-// |out_error|: If |path| cannot be created, this is set to |kSbFileInvalid|.
-//   Otherwise, it is |NULL|.
-SB_EXPORT SbFile SbFileOpen(const char* path,
-                            int flags,
-                            bool* out_created,
-                            SbFileError* out_error);
-
-// DEPRECATED with SB_API_VERSION 16
-//
-// Closes |file|. The return value indicates whether the file was closed
-// successfully.
-//
-// |file|: The absolute path of the file to be closed.
-SB_EXPORT bool SbFileClose(SbFile file);
-
-#endif  // SB_API_VERSION < 17
+#endif
```

Comment:
This conditional needs to be removed in a follow up CL. I can do it with some more starboard cleanup once this goes in.

Reply:
ack